documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1290,6 +1304,58 @@ A value of `0` disables this behavior.
 
 |===
 
+[[mongodb-monitoring]]
+== Monitoring
```

Comment:
Where is this one coming from? There already is a section about monitoring, including the shared partial for this.

Reply:
All of the connector docs include a short monitoring section. This monitoring section is separate from the shared partial.
In the MongoDB doc, which isn't yet modularized, the monitoring section was between the deployment section and the configuration properties section. Because the refactored deployment content includes the properties section as a subtopic, I moved the monitoring section to follow the properties. Make sense?

Reply:
So missed the removed monitoring above at first, as it showed up intermingled with the other stuff in the diff.

> This monitoring section is separate from the shared partial.

This is confusing though :) Because below you _do_ import the partial. So IIUC, you simply moved the existing monitoring section (including the inclusion of the partials) to a different place, so that it is consistent with the other connector pages. Is this correct?

Reply:
Ok, got it now. I kind of forgot that we do have the overall global monitoring page _and_ the specific metrics descriptions for each connector. Should be good then. Thx for clarifying!